internal/logs-files-metric-exporter/Docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openshift/origin-release:golang-1.15 AS build
```

Comment:
remove me.  not required.  Likely you will need to make revisions to the Makefile and root Dockerfile